src/Private/Helpers.ps1

```diff
@@ -1634,8 +1634,13 @@ function ConvertFrom-PodeNameValueToHashTable
     }
 
     $ht = @{}
+    $i = 0
     foreach ($key in $Collection.Keys) {
-        $ht[$key] = $Collection[$key]
+        if (!$key) {
+            $key = ''
+        }
+        $ht[$key] = $Collection[$i]
```

Comment:
Just in case the Keys and indexes don't line up for some reason, and we fetch the wrong value, would the following be safer by using `.GetValues(...)`? 🤔 

```powershell
$ht = @{}
foreach ($key in $Collection.Keys) {
    $htKey = $key
    if (!$key) {
        $htKey = ''
    }

    $ht[$htKey] = $Collection.GetValues($key)
}
```

Reply:
LGTM, it works with $null unlike the standard `[$null]`

Reply:
Wait it returns arrays now
`{"limit":"1","":"eq(status,active)","offset":"0"}` -> `{"limit":["1"],"":["eq(status,active)"],"offset":["0"]}`

Reply:
Updated